zipline/lib/adjustment.pyx

```diff
@@ -95,12 +101,12 @@ cpdef choose_adjustment_type(AdjustmentKind adjustment_kind, object value):
         raise ValueError("Unknown adjustment type %d." % adjustment_kind)
 
 
-cpdef make_adjustment_from_indices(Py_ssize_t first_row,
-                                   Py_ssize_t last_row,
-                                   Py_ssize_t first_column,
-                                   Py_ssize_t last_column,
-                                   AdjustmentKind adjustment_kind,
-                                   object value):
+cpdef Adjustment make_adjustment_from_indices(Py_ssize_t first_row,
```

Comment:
I renamed the new one to `make_adjustment_from_indices_fused`, it is cdef only so the word `fused` never leaks back into pure python space.